### ISSUE TITLE
Minor: Repeated warnings from addIterativeLSI() and plotEmbedding()

### ISSUE THREAD

**Issue by hypercompetent** [GreenleafLab/ArchR#90]
**Attach your log file**
ArchR has a built-in logging functionality for all complex functions. You MUST attach your log file (indicated in the console output) to this issue. Just drag and drop it here.
[ArchR-addIterativeLSI-7a7238e4f7af-Date-2020-05-01_Time-18-12-25.log](https://github.com/GreenleafLab/ArchR/files/4565769/ArchR-addIterativeLSI-7a7238e4f7af-Date-2020-05-01_Time-18-12-25.log)

**Describe the bug**
After running AddIterativeLSI() or plotEmbedding(), I'm getting repeated warning messages printed to the console.

**To Reproduce**
This also occurs for me with the demo datasets. I'm attaching a notebook here with those results:
[01_explore_archr_demodata.pdf](https://github.com/GreenleafLab/ArchR/files/4565948/01_explore_archr_demodata.pdf)

**Expected behavior**
Everything except the warnings :)

**Screenshots**
Here's the console output generated by `addIterativeLSI()`:
```
Checking Inputs...
ArchR logging to : ArchRLogs/ArchR-addIterativeLSI-7a7238e4f7af-Date-2020-05-01_Time-18-12-25.log
If there is an issue, please report to github with logFile!
2020-05-01 18:12:25 : Computing Total Accessibility Across All Features, 0 mins elapsed.
2020-05-01 18:12:30 : Computing Top Features, 0.077 mins elapsed.
###########
2020-05-01 18:12:30 : Running LSI (1 of 2) on Top Features, 0.085 mins elapsed.
###########
2020-05-01 18:12:30 : Sampling Cells (N = 10000) for Estimated LSI, 0.086 mins elapsed.
2020-05-01 18:12:30 : Creating Sampled Partial Matrix, 0.086 mins elapsed.
2020-05-01 18:12:48 : Computing Estimated LSI (projectAll = FALSE), 0.392 mins elapsed.
2020-05-01 18:13:58 : Identifying Clusters, 1.55 mins elapsed.
2020-05-01 18:14:27 : Identified 5 Clusters, 2.027 mins elapsed.
2020-05-01 18:14:27 : Saving LSI Iteration, 2.027 mins elapsed.
Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”2020-05-01 18:14:42 : Creating Cluster Matrix on the total Group Features, 2.279 mins elapsed.
2020-05-01 18:14:48 : Computing Variable Features, 2.392 mins elapsed.
###########
2020-05-01 18:14:49 : Running LSI (2 of 2) on Variable Features, 2.394 mins elapsed.
###########
2020-05-01 18:14:49 : Creating Partial Matrix, 2.395 mins elapsed.
2020-05-01 18:15:06 : Computing LSI, 2.688 mins elapsed.
2020-05-01 18:16:05 : Finished Running IterativeLSI, 3.661 mins elapsed.
```

Outputs from `plotEmbedding`:
```
ArchR logging to : ArchRLogs/ArchR-plotEmbedding-7a723e0f661a-Date-2020-05-01_Time-18-26-36.log
If there is an issue, please report to github with logFile!
Getting UMAP Embedding
ColorBy = cellColData
Plotting Embedding
1 
ArchR logging successful to : ArchRLogs/ArchR-plotEmbedding-7a723e0f661a-Date-2020-05-01_Time-18-26-36.log
Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”Warning message:
“Use of `dfMean$color` is discouraged. Use `color` instead.”
```

**Session Info**
```
R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Debian GNU/Linux 9 (stretch)

Matrix products: default
BLAS:   /usr/lib/openblas-base/libblas.so.3
LAPACK: /usr/lib/libopenblasp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
 [1] grid      parallel  stats4    stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] gridExtra_2.3                     ggrastr_0.1.7                    
 [3] uwot_0.1.8                        Seurat_3.1.5                     
 [5] BSgenome.Hsapiens.UCSC.hg38_1.4.1 BSgenome_1.54.0                  
 [7] rtracklayer_1.46.0                Biostrings_2.54.0                
 [9] XVector_0.26.0                    ArchR_0.9.2                      
[11] magrittr_1.5                      rhdf5_2.30.1                     
[13] Matrix_1.2-17                     data.table_1.12.8                
[15] SummarizedExperiment_1.16.1       DelayedArray_0.12.3              
[17] BiocParallel_1.20.1               matrixStats_0.56.0               
[19] Biobase_2.46.0                    GenomicRanges_1.38.0             
[21] GenomeInfoDb_1.22.1               IRanges_2.20.2                   
[23] S4Vectors_0.24.4                  BiocGenerics_0.32.0              
[25] ggplot2_3.3.0                    

loaded via a namespace (and not attached):
 [1] Rtsne_0.15               colorspace_1.4-1         ellipsis_0.3.0          
 [4] ggridges_0.5.2           IRdisplay_0.7.0          base64enc_0.1-3         
 [7] farver_2.0.3             leiden_0.3.3             listenv_0.8.0           
[10] npsurv_0.4-0             ggrepel_0.8.2            RSpectra_0.16-0         
[13] codetools_0.2-16         splines_3.6.1            lsei_1.2-0              
[16] IRkernel_1.0.2           jsonlite_1.6.1           Cairo_1.5-12            
[19] Rsamtools_2.2.3          ica_1.0-2                cluster_2.1.0           
[22] png_0.1-7                sctransform_0.2.1        compiler_3.6.1          
[25] httr_1.4.1               assertthat_0.2.1         lazyeval_0.2.2          
[28] htmltools_0.4.0          tools_3.6.1              rsvd_1.0.3              
[31] igraph_1.2.5             gtable_0.3.0             glue_1.4.0              
[34] GenomeInfoDbData_1.2.2   RANN_2.6.1               reshape2_1.4.4          
[37] dplyr_0.8.5              rappdirs_0.3.1           Rcpp_1.0.4.6            
[40] vctrs_0.2.4              gdata_2.18.0             ape_5.3                 
[43] nlme_3.1-140             lmtest_0.9-37            stringr_1.4.0           
[46] globals_0.12.5           lifecycle_0.2.0          irlba_2.3.3             
[49] gtools_3.8.2             XML_3.99-0.3             future_1.17.0           
[52] zlibbioc_1.32.0          MASS_7.3-51.4            zoo_1.8-7               
[55] scales_1.1.0             RColorBrewer_1.1-2       reticulate_1.15         
[58] pbapply_1.4-2            stringi_1.4.6            caTools_1.18.0          
[61] repr_1.0.1               rlang_0.4.5              pkgconfig_2.0.3         
[64] bitops_1.0-6             evaluate_0.14            lattice_0.20-38         
[67] ROCR_1.0-7               purrr_0.3.4              Rhdf5lib_1.8.0          
[70] labeling_0.3             GenomicAlignments_1.22.1 patchwork_1.0.0         
[73] htmlwidgets_1.5.1        cowplot_1.0.0            tidyselect_1.0.0        
[76] RcppAnnoy_0.0.16         plyr_1.8.6               R6_2.4.1                
[79] gplots_3.0.3             pbdZMQ_0.3-3             pillar_1.4.3            
[82] withr_2.2.0              fitdistrplus_1.0-14      survival_3.1-12         
[85] RCurl_1.98-1.2           tibble_3.0.1             future.apply_1.5.0      
[88] tsne_0.1-3               crayon_1.3.4             uuid_0.1-4              
[91] KernSmooth_2.23-15       plotly_4.9.2             digest_0.6.25           
[94] tidyr_1.0.2              munsell_0.5.0            viridisLite_0.3.0   
```

**Additional context**

**Comment by jgranja24** [comment]
Thanks for the post! I will look into it. I know where this occurs,
and with the new ggplot2 version now it puts a warning.

**Comment by jgranja24** [comment]
This should now be fixed in 0.9.3 version thanks!
Please re-open the issue if you have further difficulties.
